src/Response.php

```diff
@@ -196,9 +196,6 @@ private function normalizeCookieParamName($rawName)
     private function getParser()
     {
         $format = $this->getFormat();
-        if ($format === null) {
```

Comment:
Why?

Reply:
getFormat() never returns null, code inside if statement can never be reached